src/view/viewelementdefinition.jsdoc

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module engine/view/viewelementdefinition
+ */
+
+/**
+ * An object defining view element used for defining elements for conversion.
+ *
+ * @typedef {Object} module:engine/view/viewelementdefinition~ViewElementDefinition
```

Comment:
I don't know if it wouldn't be easier to define here that an element may be defined by a single string. Otherwise, everywhere this is used you'd need to write:

```
@property {String|module:engine/view/viewelementdefinition~ViewElementDefinition}
```

And you'll need to check the type of this property before deciding whether to use `ViewElement.createFromDefinition()` or the constructor (if a string was passed).